clap_builder/src/text_provider.rs

```diff
@@ -0,0 +1,48 @@
+//! Utilities for loading help texts from external sources.
```

Comment:
Simple implementation of what the API for allowing consumers to pass in their own texts might look like.